reproductions/schema/json-schema.tsx

```diff
@@ -15,34 +15,64 @@ function createValidator(schema: object) {
 const schema = {
   type: 'object',
   properties: {
-    a: {
-      type: 'number',
-      title: 'a',
-      uniforms: { title: 'Horse' },
+    regularRequired: {
```

Comment:
I know that the current example schema is not great, but I don't think that this one is any better.

Reply:
Yeah, I was using it to test visually if the validation works and the fields get highlighted. It just covers some "with dot names" fields.

Should I come up with some better schema or just revert it to the previous one?

Reply:
A better schema would be best. Maybe we should copy the one from the playground?